content/docs/command-reference/exp/pull.md

```diff
@@ -0,0 +1,65 @@
+# exp pull
+
+Download a single [experiment](/doc/command-reference/exp) from a Git remote, and
+its data from a `dvc remote`.
+
+## Synopsis
+
+```usage
+usage: dvc experiments pull [-h] [-q | -v] [-f] [--no-cache]
+                            [-r <name>] [-j <number>] [--run-cache]
+                            git_remote experiment
+
+positional arguments:
+  git_remote          Git remote name or repo URL
+  experiment          Experiment to pull
+```
+
+## Description
+
+The `dvc exp push` and `dvc exp pull` commands are the means for sharing
+experiments across <abbr>repository</abbr> copies via Git (and DVC) remotes.
+
+> Plain `git push` and `git pull` don't work with `dvc experiments` because
+> these are saved under custom Git references. See **How does DVC track
```

Comment:
Is there an equivalent `git` command this is a "wrapper" of? Like `git pu* --refs exps` or something like that.

Reply:
It's just equivalent to `git push` and `git fetch` using the full refnames for experiment refs. The only reason a user can't use `git push` and `git fetch` out of the box is because git will not resolve the shortened experiment name into a full `refs/.../<refname>` path - git only does that automatically for `refs/tags` and `refs/heads` (branches).

Reply:
If you want to get specific, `exp pull` is equivalent to

```
$ git fetch [-f/--force] <git_remote> refs/exps/<internal_exp_hash>/<exp_name>:refs/exps/<internal_exp_hash/<exp_name>
```

Reply:
`exp pull` is technically also an explicit `git fetch` rather than `git pull` (git pull is a two part fetch + merge operation that fetches into one reference and then merges it into a second one). `exp pull` does not have a merge step.

Reply:
> equivalent to git push and git fetch

Hey yeah, why not `exp fetch`? (note for self for now)

Thanks for all the details @pmrowla looks like you're quite the Git expert now!